NWNXBase.h

```diff
@@ -114,6 +125,7 @@ class CNWNXBase
     gline* nwnxConfig;
     const char * confKey;
     int debuglevel;
+    int logTimestamp;
```

Comment:
Do you expect it to be anything other than true/false? (Like additional datetime formats)